deploy/lynx.yaml

```diff
@@ -0,0 +1,2689 @@
+                  - externalIDValue
+                  type: object
+                ips:
+                  items:
+                    description: IPAddress is net ip address, can be ipv4 or ipv6.
+                      Format like 192.168.10.12 or fe80::488e:b1ff:fe37:5414
+                    pattern: ^(((([1]?\d)?\d|2[0-4]\d|25[0-5])\.){3}(([1]?\d)?\d|2[0-4]\d|25[0-5]))|([\da-fA-F]{1,4}(\:[\da-fA-F]{1,4}){7})|(([\da-fA-F]{1,4}:){0,5}::([\da-fA-F]{1,4}:){0,5}[\da-fA-F]{1,4})$
+                    type: string
+                  type: array
+              required:
+              - endpointReference
+              type: object
+            type: array
+        required:
+        - appliedToGroupMembers
+        type: object
+    served: true
+    storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: policyrules.policyrule.lynx.smartx.com
+spec:
+  group: policyrule.lynx.smartx.com
+  names:
+    kind: PolicyRule
+    listKind: PolicyRuleList
+    plural: policyrules
+    singular: policyrule
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: PolicyRule
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: PolicyRuleSpec defines the desired state of PolicyRule
+            properties:
+              action:
+                type: string
+              defaultPolicyRule:
+                type: boolean
+              direction:
+                type: string
+              dstIpAddr:
+                type: string
+              dstPort:
+                type: integer
+              dstPortMask:
+                type: integer
+              ipProtocol:
+                type: string
+              srcIpAddr:
+                type: string
+              srcPort:
+                type: integer
+              srcPortMask:
+                type: integer
+              tcpFlags:
+                type: string
```

Comment:
Please refer to https://github.com/smartxworks/lynx/commit/fe055ad93a5d60714d54ab4273cd422f09ef754b and keep the same style.

Reply:
This yaml file was generated automatically, it was consist of all other yaml files.